src/pages/Contact.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { useParams, Link, useOutletContext } from 'react-router-dom';
+import DeleteConfirmation from '../components/DeleteConfirmation';
+
+const Contact = () => {
```

Comment:
Great work on your project structure - I like the pages vs components distinction, and mirrors how the app works.